samples/subsys/ipc/openamp/sample.yaml

```diff
@@ -4,6 +4,7 @@ sample:
 tests:
   sample.ipc.openamp:
     filter: dt_chosen_enabled("zephyr,ipc") and dt_chosen_enabled("zephyr,ipc_shm")
+      and not (CONFIG_SOC_ESP32_APPCPU or CONFIG_SOC_ESP32S3_APPCPU)
```

Comment:
Another option would be to `platform_exclude` all esp32 and esp32s3-based appcpu boards. As all those board sets `_APPCPU`, this sounds good. Any other suggestion is welcome.